apps/frontend/src/components/UserConfigDialog.vue

```diff
@@ -244,7 +242,7 @@
     };
     const sendNewStatus = async () => {
         const { sendSocketUserStatus } = useSocketActions();
-        sendSocketUserStatus(userStatus.value);
+        await sendSocketUserStatus(userStatus.value);
```

Comment:
I am not sure if await is needed since it will only slow down the application and we do not need to wait for all our contacts to have updated our status